### ISSUE TITLE
@apollographql/graphql-playground-html generates incorrect links if you pass a falsey version

### ISSUE THREAD

**Issue by silenaker** [apollographql/apollo-server#4937]
1.6.26 has a problem, that is
if you don't want a fixed version number and set version option to "", you will get a wrong url

```typescript
const apolloServer = new ApolloServer({
  schema,
  context: fastify,
  playground: {
    version: "",
    cdnUrl: "",
    settings: {
      "editor.theme": "light",
    },
  },
});

fastify.register(fastifyStatic, {
  root: path.join(__dirname, "../node_modules/@apollographql/graphql-playground-react/"),
  prefix: "/@apollographql/graphql-playground-react/"
});
```
will contribute to
```html
<link rel="stylesheet" href="/@apollographql/graphql-playground-reactbuild/static/css/index.css">
```

I found a workaround for the moment，you must set the version option and set route correctly 

```typescript
const apolloServer = new ApolloServer({
  schema,
  context: fastify,
  playground: {
    version: "1.6.26", // must set version
    cdnUrl: "",
    settings: {
      "editor.theme": "light",
    },
  },
});

fastify.register(fastifyStatic, {
  root: path.join(__dirname, "../node_modules/@apollographql/graphql-playground-react/"),
  prefix: "/@apollographql/graphql-playground-react@1.6.26/", // must use corresponding version
});
```

1.6.29 has fixed this bug

**Comment by glasser** [comment]
I'm not sure I follow. I'm looking at https://github.com/graphql/graphql-playground/commits/master/packages/graphql-playground-html and I don't see any commits between 1.6.26 and 1.6.29 that seem relevant here?

**Comment by glasser** [comment]
Also confused because our fork appears to hardcode that `/build`  in the stylesheet link? https://github.com/apollographql/graphql-playground/blob/e598f561eace3d0cfecc39b08b1abd3a0a207b5e/packages/graphql-playground-html/src/render-playground-page.ts#L78

**Comment by glasser** [comment]
Ah, hmm, that code on master in our fork does not appear to match what is published to npm.

**Comment by glasser** [comment]
Ah, gotcha, I should be looking at the `apollo` branch rather than the `master` branch.

**Comment by glasser** [comment]
As a workaround, if you don't pass `version` at all then Apollo Server will select a version for you and that version should work, though perhaps it might not be the version you want?

**Comment by silenaker** [comment]
seems that the auto version selection did not work if i did't pass the version option

**Comment by abernix** [comment]
@silenaker Note my comment in https://github.com/apollographql/apollo-server/pull/4932#issuecomment-785004544.  If you can update your transitive dependencies (possibly by forcing use of `@apollographql/graphql-playground-html` version `1.6.27` via your yarn or package lock files, you may be able to test this, otherwise, we'll get a new version published soon.

**Comment by glasser** [comment]
I believe this is fixed by https://github.com/apollographql/apollo-server/pull/4955

**Comment by glasser** [comment]
I've published a release with this to version `2.21.1-alpha.0`. My plan is to release 2.21.1 on Monday. If you want to test that this fix works before then, try running the prerelease yourself, and provide feedback on https://github.com/apollographql/apollo-server/pull/4990
